cmd/root.go

```diff
@@ -342,6 +335,21 @@ var rootCmd = &cobra.Command{
 			panic(err)
 		}
 
+		var ciiClient clients.CIIBestPracticesClient
+		var ossFuzzRepoClient clients.RepoClient
+
+		if _, exists := enabledChecks[checks.CheckCIIBestPractices]; exists {
+			ciiClient = clients.DefaultCIIBestPracticesClient()
+		}
+
+		if _, exists := enabledChecks[checks.CheckCIIBestPractices]; exists {
+			ossFuzzRepoClient, err = githubrepo.CreateOssFuzzRepoClient(ctx, logger)
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer ossFuzzRepoClient.Close()
+		}
```

Comment:
Instead of basing this on whats enabled, we can use repoType. Basically, update `getRepoAccessors` to return both CII and OssFuzz clients.

Reply:
So in case of `local` these will be `nil` irrespective of which check is enabled.

Reply:
does it help to do that? These are actually needed unless they're enabled, right?